### ISSUE TITLE
When are safe operators enabled by default?

### ISSUE THREAD

**Issue by jellelicht** [noctuid/lispyville#37]
Unless I am somehow misunderstanding a major part of lispyville and how it can be used, I am having some issues getting the operators such as safe delete to work properly.

If we have, line-by-line `C-M-x`'ed:
```
(require 'evil) ; => evil
(require 'lispyville) ; => lispyville
(evil-mode) ; => t
(lispyville-mode) ; => t

(defun inc (n)
 | (+ 1 n))
```
With `|` being the cursor/point in normal mode, pressing `D` deletes the whole line (including `))`, leaving us in an inconsistent state:
```
(defun inc (n)
|
```

Am I somehow misconfiguring lispyville, or am I misunderstanding what should be happening? Calling e.g. `lispyville-delete-whole-line` via `M-x` does lead to a consistent state, where the last `)` is kept.

**Comment by jellelicht** [comment]
Nvm, I figured it out; the last release of `evil` barely predates a commit (by you even, if I'm not mistaken) that slightly changes `evil-define-key*`. Sorry for the noise, and thanks for the amazing package!

**Comment by noctuid** [comment]
Right, I should probably make it more clear in the readme that this package tracks the latest evil.
